learningresources/tests/test_views.py

```diff
@@ -19,16 +21,14 @@
 class TestViews(LoreTestCase):
     """Hit each view."""
 
+    repository_url = "/repositories/{0}/"
```

Comment:
Same comment here as above, about putting it in setup and having `self.repo.slug` in there once.